### DISCUSSION TITLE
2.0.0-beta now available as pre-release

### DISCUSSION THREAD

**Discussion by gkoh** [gkoh/furble#82] [Announcements]
2.0.0 is now in beta and available as a pre-release, details are here: https://github.com/gkoh/furble/releases/tag/v2.0.0-beta

New in this version are:
* GPS location tagging on Fujifilm with the supported M5Stack GPS unit
* support for M5StickC Plus2
* basic support for M5Stack Core/Core2
* shutter lock
* intervalometer
* a "fancy" new web installer

**Comment by gkoh** [comment]
2.0.0-rc3 is now in pre-release with support for unique identifiers in case you have more than one remote.

**Comment by gkoh** [comment]
2.0.0-rc4 is now in pre-release with:
* fixed GPS on M5Stack Core(2)
* non-flickering backlight during idle on M5StickC Plus2 and Core

**Comment by gkoh** [comment]
2.0.0 is now released, identical to v2.0.0-rc4 as no further changes appeared to be required.

**Reply by wolcano** [reply]
Configuration:
- M5StickC Plus 1.1 with v2.0.0-beta+1 (using web installer, Chrome)
- Canon EOS RP (wireless communication set to "Bluetooth function: Smartphone")

Working:
- Connect
- (auto-reconnect to camera after a while)
- Interval
- Shutter: Release
- Backlight
- Theme

Not working:
- Shutter: Focus (does nothing)

Did not test:
- scan (I reused the saved connection from previous installation)
- GPS

**Reply by gkoh** [reply]
Thanks for testing!

> Not working:
>    * Shutter: Focus (does nothing)

This is a known issue as the bug is still open. (Hmm, I should probably create a table of features vs platform for the README).
@wolcano if you are still available to help test, I can try and refresh #38?
